src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/sentencing/SentencingService.kt

```diff
@@ -1348,7 +1348,10 @@ class SentencingService(
       throw BadDataException("Cannot clone court cased from the latest booking $bookingId on to itself")
     }
 
-    val clonedCasesWithSource = booking.courtCases.map { sourceCase ->
+    return cloneCourtCasesToLatestBookingFrom(latestBooking, booking.courtCases)
+  }
+  private fun cloneCourtCasesToLatestBookingFrom(latestBooking: OffenderBooking, sourceCourtCases: List<CourtCase>): BookingCourtCaseCloneResponse {
```

Comment:
Extracted to a function since next step will be to only clone cases related to a specific case rather than all cases in a booking